ui/app/routes/vault/cluster/secrets/backend/configuration/edit.ts

```diff
@@ -66,6 +68,16 @@ export default class SecretsBackendConfigurationEdit extends Route {
         }
       }
     }
+    // if the type is AWS and it's enterprise, we also fetch the issuer
```

Comment:
Might be worth mentioning in the comment that fetching the issuer is not on the aws config api, but on a global endpoint with no associated model/adapter.